README.md

```diff
@@ -78,12 +89,6 @@ In a .NET application, you install a Sentry package using NuGet and use it to se
 
 In a Ruby application, you install the Sentry gem and configure it to add additional context. You can find an example of setting this up in the [Income API][ruby-sentry-example].
```

Comment:
This can be made generic as Sentry is available in most languages.

Reply:
This mentions ruby and .net so covers our two main languages in use. Going to leave it as-is for now because this was already in the repo